src/sp/jsp_cl.cpp

```diff
@@ -841,7 +841,16 @@ jsp_create_stored_procedure (PARSER_CONTEXT *parser, PT_NODE *statement)
 
   if (sp_info.lang == SP_LANG_PLCSQL)
     {
-      compile_request.code.assign (statement->sql_user_text, statement->sql_user_text_len);
+      if (statement->sql_user_text && statement->sql_user_text_len > 0)
+	{
+	  compile_request.code.assign (statement->sql_user_text, statement->sql_user_text_len);
```

Comment:
```suggestion
	  const char *plcsql_text = parser_print_tree (parser, statement);
	  compile_request.code.assign (plcsql_text, strlen (plcsql_text));
```

오프라인 리뷰에서와 같이 위와 같이 하나의 로직으로만 코드가 저장되면 좋을 것 같습니다.
이 과정으로 unloaddb에서도 다른 객체와 일관성을 맞출 수 있을 것 같습니다.

Reply:
에러 라인 수에 변경이 있을 수 있어서 sql_user_text (사용자 입력 그대로) 가 loaddb 때도 non-null 이 되도록 다시 구현했습니다.